### PR TITLE
Delete posts

### PR DESCRIPTION
**Pull request by danielgrijalva**
## Pull Request
This lets users delete their own posts/contributions. 

- Created post deletion modal
- Added post deletion locales (EN and ES)
- Added "Delete post" option in contribution menu
- Redirects to `/` _or_ refreshes page after deleting a post

human connection delete posts demo

### Issues

- [X] Resolves issue #330 

### Todo

- [ ] Tests pending. They should be ready next week.
- [ ] Translations. Only english and spanish were provided.

### CODE REVIEW

**Comment by roschaefer** [comment]
@danielgrijalva @Tirokk I just resolved merge conflicts through Github.

We :heart: you @danielgrijalva

So much love

As you mentioned, tests are still missing. Translations are fine, we can add them through [lokalise](https://lokalise.co/project/556252725c18dd752dd546.13222042/).

@danielgrijalva @Tirokk would one of you be willing to contribute some frontend tests?

**Comment by Tirokk** [comment]
@roschaefer Best is @danielgrijalva write the Frontend tests himself. He seems to be smart to find out.

@danielgrijalva If you need help to write Frontend tests, please contact me ( @Tirokk ).
@mattwr18 Has a lot of experience in Frontend test writing too …

**Comment by Tirokk** [comment]
Are you on our (Human Connection Discord)[https://discord.gg/6ub73U3] chat @danielgrijalva ?

Would be nice to see you there! Please contact me (Wolle (Hamburg)) there, if you can.

**Comment by danielgrijalva** [comment]
@Tirokk I'd be glad to provide the tests! 😄 but it'll take some more time... you see, I injured my right hand so I won't be able to code for a week or so. I'll join the discord and contact you as soon as I can 🙆‍♂

**Comment by Tirokk** [comment]
Thanks for response @danielgrijalva !

I wish you a good recovery and hope that it hurts not to much and your hand gets quick well again.
Take the time you need and I am happy to hear you.
See you on Discord.

File: webapp/components/Modal/DeleteModal.spec.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    const Wrapper = () => {
+      return mount(DeleteModal, { propsData, mocks, localVue, router})
+    }
+
+    beforeEach(jest.useFakeTimers)
+
+    it('renders', () => {
+      expect(Wrapper().is('div')).toBe(true)
+    })
+
```
[the remark this answers is not in the document]
Comment (roschaefer) [reply webapp/components/Modal/DeleteModal.spec.js]: @mattwr18 sure, usually I start off by writing this kind of test case only to make sure I have everything set in place to start the component test. Later, I just don't remove it. I won't miss it so badly, so go ahead removing it.

**Comment by mattwr18** [comment]
@roschaefer @danielgrijalva is in UTC -7, so I think the meetings would be in the middle of the night for him.

We have talked about having some pairing sessions around `9pm UTC`, but so far we haven't hooked up.

As I stated :point_up:, I also think it's a great job, and was giving suggestions, not blocking it being merged with requested changes. :smile:

**Comment by ulfgebhardt** [comment]
@roschaefer is here a problem regarding travis since the branch is on a foreign repo? If yes i would be very interested in the solution

**Comment by roschaefer** [comment]
@ulfgebhardt I observed the same: If we receive PR from `origin`<-`fork` then the `Travis CI Branch` check is pending indefinitely. I have 0 idea why. Could you do some research?

**Comment by mattwr18** [comment]
Maybe, irrelevant, but just as a reminder, and also for @ulfgebhardt who wasn't around on a daily basis when we had the discussion. Any encrypted variables in Travis are not accessible by `forks`.

Again, maybe not relevant as I think @ulfgebhardt you said that we are not using `$CODE_COV`, but at the time we had the discussion about encouraging people not to fork and put in `PR`s, but to ask, and we would happily give them write access.

**Comment by ulfgebhardt** [comment]
i will just push this on our repo and merge - noting the research task in order to create a ticket

**Comment by ulfgebhardt** [comment]
@roschaefer the only solution to this travis bug is to disable branch build (we should discuss this) - a workaround is to finish everything in the forks pr and then push the branch to the HC Repo. The approves will be kept. Can be seen here: https://github.com/Human-Connection/Human-Connection/pull/595

Ressource: https://travis-ci.community/t/travis-ci-branch-expected-waiting-for-status-to-be-reported/829

**Pull request merged**